packages/fxa-auth-db-mysql/lib/db/mem.js

```diff
@@ -1086,6 +1086,41 @@ module.exports = function (log, error) {
     }).reverse())
   }
 
+  Memory.prototype.securityEventsByUid = function (uid) {
+    if (! uid) {
+      return P.reject(error.notFound())
+    }
+    const key = uid.toString('hex')
+    const events = securityEvents[key] || []
+    const securityEventsLimit = 50
```

Comment:
Lets pull this to the top of the file similar to https://github.com/mozilla/fxa/blob/ebd32d7eccd804d8a9ef359f04b15ac9509811f1/packages/fxa-auth-db-mysql/lib/db/mem.js#L57.

I think we can make this limit 100.